src/colvarcomp_combination.cpp

```diff
@@ -226,97 +226,107 @@ colvar::customColvar::~customColvar() {
 #endif
 }
 
+
 void colvar::customColvar::calc_value() {
+    if (!use_custom_function) {
+        colvar::linearCombination::calc_value();
+    } else {
 #ifdef LEPTON
-    for (size_t i_cv = 0; i_cv < cv.size(); ++i_cv) {
-        cv[i_cv]->calc_value();
-    }
-    x.reset();
-    size_t l = 0;
-    for (size_t i = 0; i < x.size(); ++i) {
         for (size_t i_cv = 0; i_cv < cv.size(); ++i_cv) {
-            const colvarvalue& current_cv_value = cv[i_cv]->value();
-            for (size_t j_elem = 0; j_elem < current_cv_value.size(); ++j_elem) {
-                if (current_cv_value.type() == colvarvalue::type_scalar) {
-                    *(value_eval_var_refs[l++]) = cv[i_cv]->sup_coeff * (cvm::pow(current_cv_value.real_value, cv[i_cv]->sup_np));
-                } else {
-                    *(value_eval_var_refs[l++]) = cv[i_cv]->sup_coeff * current_cv_value[j_elem];
+            cv[i_cv]->calc_value();
+        }
+        x.reset();
+        size_t l = 0;
+        for (size_t i = 0; i < x.size(); ++i) {
+            for (size_t i_cv = 0; i_cv < cv.size(); ++i_cv) {
+                const colvarvalue& current_cv_value = cv[i_cv]->value();
+                for (size_t j_elem = 0; j_elem < current_cv_value.size(); ++j_elem) {
+                    if (current_cv_value.type() == colvarvalue::type_scalar) {
+                        *(value_eval_var_refs[l++]) = cv[i_cv]->sup_coeff * (cvm::pow(current_cv_value.real_value, cv[i_cv]->sup_np));
+                    } else {
+                        *(value_eval_var_refs[l++]) = cv[i_cv]->sup_coeff * current_cv_value[j_elem];
+                    }
                 }
             }
+            x[i] = value_evaluators[i]->evaluate();
         }
-        x[i] = value_evaluators[i]->evaluate();
-    }
+#else
+        cvm::error("Try to use customColvar with customFunction but without Lepton compiled.\n");
```

Comment:
Here I would tweak the wording to make clearer whether you are describing what went wrong with the user's input vs. recommending what to do instead. Also, it would be appropriate to use `COLVARS_INPUT_ERROR` as second argument to the error function.